common/djangoapps/course_modes/models.py

```diff
@@ -375,6 +403,128 @@ def min_course_price_for_currency(cls, course_id, currency):
         modes = cls.modes_for_course(course_id)
         return min(mode.min_price for mode in modes if mode.currency == currency)
 
+    @classmethod
+    def get_certificate_display(cls, mode, verification_status):
```

Comment:
Technically, this isn't the certificate display. It's the enrollment mode display. There is an actual area where the certificate information is displayed on the dashboard, and this naming might get confusing.